apps/services/endorsements/api/README.md

```diff
@@ -0,0 +1,66 @@
+# Endorsement API
+
+## About
+
+This service manages endorsement lists within island.is.
+
+## Initial setup
+
+### Prerequisites
+
+To run the API locally make sure you:
+
+- Have [Docker](https://www.docker.com/products/docker-desktop) running
+- Fulfill all prerequisites listed in:
+- - [Getting started section at docs.devland.is](https://docs.devland.is/)
+- - [AWS secrets section at docs.devland.is](https://docs.devland.is/repository/)
+
+### Environment variables
+
+[direnv](https://direnv.net/docs/installation.html) is recommended for autoloading environment variables
+
+-- add this line to your ~/.zshrc or equivalent
+
+```bash
+eval "$(direnv hook zsh)"
+```
+
+once done you should see all variables autoload everytime you cd into the "island.is" folder
+
+### Initalize the application
+
+```bash
+yarn nx run application-templates-party-application:dev/init
+```
+
+## Running locally
+
+Sign into AWS
+
+```bash
+aws sso login
+```
+
+Then run application using this command
+
+```bash
+yarn nx run application-templates-party-application:dev
```

Comment:
Is the `dev` workspace task leaking processes with the use of `&` in the commands?

Reply:
This can create zombies in some cases but generally these processes should stop when the process is stopped.
We ended up doing this since as this is likely to work in all environments.
There might be a better approach to this, but this has been working without issues for the team working on this since added